console/src/main/scripts/plugins/metrics/ts/directives/overview/statusOverviewDirective.ts

```diff
@@ -20,6 +20,28 @@
 
 module HawkularMetrics {
   export class StatusOverviewController {
+    constructor(private $filter: ng.IFilterService) {}
+
+    public parseDayFromTimestamp(timestamp): string {
+      const dateFilter = this.$filter('date');
+      const parsedDate = new Date(timestamp);
+      const today = new Date();
+      const yesterday = new Date();
+      yesterday.setDate(today.getDate() - 1);
+      if (StatusOverviewController.isSameDayMonthYear(parsedDate, yesterday)) {
```

Comment:
Yeah I guess this might be better in moment.js
I will change it and reopen PR once egain.